usr/lib/icsf_stdll/pbkdf.c

```diff
@@ -371,10 +410,28 @@ CK_RV pbkdf(CK_BYTE * password, CK_ULONG len, CK_BYTE * salt, CK_BYTE * dkey,
     }
 
 out:
+    if (rc == CKR_OK && tokdata != NULL &&
+        (tokdata->statistics->flags & STATISTICS_FLAG_COUNT_INTERNAL) != 0) {
+        tokdata->statistics->increment_func(tokdata->statistics,
+                                            tokdata->slot_id, &mech,
+                                            POLICY_STRENGTH_IDX_0);
+        if ((tokdata->statistics->flags & STATISTICS_FLAG_COUNT_IMPLICIT) != 0) {
+            /*
+             * We use CKM_PKCS5_PBKD2 with CKP_PKCS5_PBKD2_HMAC_SHA256.
+             * Use strength 0 because the HMAC key is the pin, and it is max
+             * 8 char (i.e. 64 bit) long, which is way below 112 bit anyway.
```

Comment:
Technically that would be up to whoever configures the strength.conf-file.

Reply:
I think we already discussed this offline.... The conclusion was that it makes absolutely no sense to map this to anything higher than or equal to 112 bits strength (although it is technically possible to do so). So we simply assume below 112 bits and thus always use strength zero.